### PR TITLE
odc docs pages add

### PR DESCRIPTION
**Pull request by darkblue-b**

### CODE REVIEW

**Comment by darkblue-b** [comment]
need to add these assets ?
```

OSGEO/OSGeoLive-doc$ grep _odc doc/overview/odc_overview.rst 
@LOGO_odc@
@OSGEO_KIND_odc@
@NAME_odc@
@SCREENSHOT_odc@
**Website:** @WEB_odc@
@VMDK_odc@
@QUICKSTART_odc@

OSGEO/OSGeoLive-doc$ grep _odc doc/quickstart/odc_quickstart.rst
@LOGO_odc@
@OSGEO_KIND_odc@
@VMDK_odc@
@NAME_odc@ Quickstart
* Website: @WEB_odc@

```

**Comment by kalxas** [comment]
I see most of the quickstart is about gmt, is this work in progress?

**Comment by cvvergara** [comment]
> need to add these assets? 

Its not processing that new documentation
* add it to `projects_info.csv`  (all the other assets mentioned in your question)
* Once its added you will get errors because of  @SCREENSHOT_odc@
  * You need to add the file: `odc_screenshot.png` (same directory as the logo)

File: doc/quickstart/odc_quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  *  python3-pystac
+
+* `TODO tutorial <https://docs.generic-mapping-tools.org/latest/tutorial>`
+
+What next?
+==========
+
+* Documentation: http://datacube-core.readthedocs.io/en/latest/
+* Website: @WEB_odc@
+* Support: Slack <http://slack.opendatacube.org>
```
Comment (astroidex) [inline doc/quickstart/odc_quickstart.rst:126]: link without <>

File: doc/overview/odc_overview.rst [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+**Licence:** Apache License Version 2
+
+**Software Version:** |version-odc|
+
+**Supported Platforms:** GNU/Linux, Mac OSX, MS Windows
+
+**API Interfaces:** Command line
+
+**Support:** https:///
```
Comment (astroidex) [inline doc/overview/odc_overview.rst:64]: add Slack link too Slack http://slack.opendatacube.org

**Review by astroidex — CHANGES_REQUESTED** [review]

File: doc/overview/odc_overview.rst (hunk 2 above)
Comment (darkblue-b) [reply doc/overview/odc_overview.rst]: revise PR; add screenshot, alpha text

**Review by cvvergara — APPROVED** [review]

**Pull request merged**